File: Question4/Question4.cs
using System.CodeDom.Compiler;
using System.Collections.Generic;
using System.Collections;
using System.ComponentModel;
using System.Diagnostics.CodeAnalysis;
using System.Globalization;
using System.IO;
using System.Linq;
using System.Reflection;
using System.Runtime.Serialization;
using System.Text.RegularExpressions;
using System.Text;
using System;

class Solution {

    // Complete the bankersAndClients function below.
    static int totalPairs(int n, List<int> values) {
        return -1;

    }

    static void Main(string[] args) {

		int n = Convert.ToInt32(Console.ReadLine().Trim());
		List<int> values = Console.ReadLine().TrimEnd().Split(' ').ToList().Select(aTemp => Convert.ToInt32(aTemp)).ToList();

        int result = totalPairs(n, values);
        // Do not remove below line
		Console.WriteLine(result);
    }
}

File: Question4/Question4.java
import java.io.*;
import java.math.*;
import java.security.*;
import java.text.*;
import java.util.*;
import java.util.concurrent.*;
import java.util.regex.*;

class Solution {

    static int totalPairs(int n, int[] values) {
        // Complete the total Pairs function below
        return -1;

    }

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        int[] values = new int[n];
        String[] temp = br.readLine().split(" ");
        for (int i = 0; i < n; ++i) {
            values[i] = Integer.parseInt(temp[i]);
        }

        int answer = totalPairs(n, values);
        System.out.println(answer);
    }
}
